src/inline-plugin.js

```diff
@@ -0,0 +1,45 @@
+(function() {
+    tinymce.PluginManager.add('inline_tinymce_plugin', function(editor, url) {
+        editor.addButton('inline_tinymce_button', {
+            title: 'Click to add in-line text',
+            text: 'Add In-line',
+            icon: false,
+            onclick: function() {
+                var text = editor.selection.getContent({ format: 'html' });
+                if (!text) {
+                    editor.windowManager.alert('Please highlight some text before adding in-line.');
+                    return;
+                }
+                editor.windowManager.open({
+                    title: 'In-line Commentary',
+                    body: [
+                        {
+                            type: 'textbox',
+                            name: 'EPR',
+                            label: 'Name of Reviewer:',
+                        },
+                        {
+                            type: 'textbox',
+                            multiline: true,
+                            name: 'inline',
+                            label: 'Commentary:',
+                        },
+                    ],
+                    onsubmit: function(e) {
+                        editor.execCommand(
+                            'mceReplaceContent',
+                            false,
+                            '<span class="inline" data-EPR="' +
```

Comment:
You're going to need to either escape this better or use a different method of inserting the html element.

Currently, any text starting with a `"` will break this. Also, I'm able to insert `<script>runSomeBadStuff()</script>` because there's really no sanitation (which is less important for us, but more important if you want to publish this publicly).